### PR TITLE
Feature/lu 4025 validation pipeline

### PR DESCRIPTION
**Pull request by ranjit-das**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+**Quality-checks:**
+
+The quality checks contain all the jobs for linting and testing the pull requests which are raised for each pipeline. These are all triggered when a pull request is raised in a respective repository.
+
+**Deployment:**
+
+This section contains all of the server deployment jobs of the pipeline. This are  triggered when a merge is made to the dev branch. This way our AWS code is always up to date with what is on the dev branch. At later point, during release, Live environment's AWS code will be up to date with master branch.
+
+**Steps** <br/>
+Please follow the steps given below -
```
Comment (RyanAldy) [inline README.md]: What about the step for setting up the target?

File: initial-pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    source:
+      repository: python
+      tag: 3.7-slim
+
+### ECR bits
+  - name: ecr-image-validation-save
+    type: docker-image
+    icon: docker
+    source:
+      repository: 014669633018.dkr.ecr.eu-west-2.amazonaws.com/takeon-cicd
```
Comment (RyanAldy) [inline initial-pipeline.yml]: Shouldn't we set up an ECR for this lambda specifically?

File: README.md (hunk 1 above)
Comment (ranjit-das) [reply README.md]: Updated README accordingly.

File: initial-pipeline.yml (hunk 2 above)
Comment (ranjit-das) [reply initial-pipeline.yml]: At the moment using common ECR repo 'takeon-validation-cicd' and tagging each lambda under it e.g. 'evaluator', 'save' etc.

**Pull request merged**